pubspec.yaml

```diff
@@ -11,7 +11,6 @@ environment:
 
```

Comment:
Please bump the version to 0.4.0+1 (if you want to publish now) or 0.4.1-dev if you don't want to publish. And add an entry to the changelog